package.json

```diff
@@ -1,10 +1,12 @@
 {
-  "name": "code-sample-editor",
-  "version": "0.1.0-pre.3",
+  "name": "playground-elements",
+  "version": "0.1.0",
   "description": "A multi-file code editor component with live preview",
+  "homepage": "https://github.com/PolymerLabs/playground-elements#readme",
+  "repository": "github:PolymerLabs/playground-elements",
   "type": "module",
-  "module": "lib/code-sample-editor.js",
-  "main": "lib/code-sample-editor.js",
+  "module": "lib/playground-ide.js",
+  "main": "lib/playground-ide.js",
```

Comment:
might want to make an index.js file so -ide doesn't always have the be the aggregating module.

Reply:
Done.